.github/workflows/tests.yml

```diff
@@ -256,13 +249,6 @@ jobs:
         if: matrix.build_type == 'system'
         run: bin/ember-cli --build
 
-      - name: Minio cache
-        if: matrix.build_type == 'system' && matrix.target == 'core'
-        uses: actions/cache@v4
-        with:
-          path: ${{ env.MINIO_RUNNER_INSTALL_DIR }}
-          key: ${{ runner.os }}-${{ steps.container-envs.outputs.ruby_version }}-${{ steps.container-envs.outputs.debian_release }}-gem-${{ hashFiles('**/Gemfile.lock') }}
```

Comment:
No longer required cause the `discourse/discourse_test:release` image already downloads this.